shared/styles/style-guide.desktop.js

```diff
@@ -52,7 +51,10 @@ export const globalColorsDZ2 = {
   lightGrey2: '#f6f6f6',
 
   white: '#ffffff',
-  black: '#000000'
+  black: '#000000',
+
+  // TODO these do not appear in the color style guide, only text mocks
+  brown: '#471f11'
```

Comment:
@cecileboucheron can you verify this is correct. if so we should add it to the color sheet or remove this